slugid/slugid.py

```diff
@@ -1,8 +1,10 @@
 # Licensed under the Mozilla Public Licence 2.0.
 # https://www.mozilla.org/en-US/MPL/2.0
 
+import sys
 import uuid
 import base64
+two = True if sys.version_info.major == 2 else False
```

Comment:
fwiw, performance is the same in python for this, but in JS, looking up members of an object is a performance drain because it has to go through the prototype chain each time.  I must have been thinking in JS when I wrote this :)